build.gradle.kts

```diff
@@ -27,9 +27,6 @@ allprojects {
         maven("https://cache-redirector.jetbrains.com/intellij-dependencies")
         // Repo for the backported Android IntelliJ Plugin by Jetbrains used in Ultimate
         maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies/")
-        maven("https://jitpack.io") {
-            content { includeGroup("com.github.requery") }
-        }
```

Comment:
Are we suggesting somewhere on the docs that users do this as well? I couldn't find anything from a quick search, but if we are we should update that too.